src/main/kotlin/inc/evil/d17map/dtos/ExtraRoomRequest.kt

```diff
@@ -12,5 +10,6 @@ class ExtraRoomRequest (
     var description: String,
     var type: String,
     val modelKey: String,
-    val floor: Floor
+    val floorName: String,
```

Comment:
Tutaj analogicznie jak wcześniej bym się zastanowił nad zmianą nazwy pola na po prostu `floor` I w przypadku budynku tak samo, ale to tylko sugestia daję wolną rękę. Ewentualnie do przegadania z frontem.